pkg/upgrade/upgrade.go

```diff
@@ -424,6 +428,11 @@ func unsetOwnerReference(ctx context.Context, cli client.Client, instanceName st
 			return fmt.Errorf("error unset ownerreference for CR %s : %w", instanceName, err)
 		}
 	}
+	// flip TrustyAI BiasMetrics to false (.spec.dashboardConfig.disableBiasMetrics)
```

Comment:
I think you still need to gate this configuration change to only execute when upgrade from version < 2.14 to version >= 2.14. This will prevent the case where a user running 2.14 (or later version) manually disables this feature, to be overriden on their next upgrade.

Reply:
i was thinking this will be a temp workaround which only lives in 2.14, then  we can remove the call in 2.15.
from downstream skipRange: (different from ODH upgrade path  which we allow on jump from 2.X to the latest release 2.17 for now)
- 2.8.x can one jump to 2.10
- 2.11 can only go from 2.10
- 2.12 can only go from 2.11
- 2.13 can only go from 2.12

Reply:
In theory, 2.8.x will also be able to jump to ~2.17 when that is released (EUS channel). 

In order to know which version the operator is upgrading from, can we use the DSCI status? https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/pkg/cluster/cluster_config.go#L133